action.yml

```diff
@@ -73,13 +73,21 @@ inputs:
     default: ''
     required: false
   deb_depends:
-    description: 'Comma separated list of deb dependencies.'
+    description: 'Comma-separated list of deb dependencies.'
     default: ''
     required: false
+  deb_recommends:
```

Comment:
💭 for the recommends and depends inputs, could we include a blurb about how these inputs affect the behaviour of the install - i.e required dependencies (package names) that will be attempted to be installed automatically and fail if can't vs will suggest/try to install but won't cause the install to fail?